File: templates/Uwp/_comp/_shared/Feat.UnitTests.Core.SampleData.Chart.NUnit._VB/Param_ProjectName.Core.Tests.NUnit/Tests_postaction.vb
'{[{
Imports Param_RootNamespace.Core.Services
'}]}

Public Class Tests
    '^^
    '{[{

    ' TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <Test>
    Public Async Sub EnsureSampleDataServiceReturnsChartDataAsync()
        Dim actual = await SampleDataService.GetChartSampleDataAsync()

        Assert.IsNotEmpty(actual)
    End Sub
    '}]}
End Class

File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.Chart.MSTest/Param_ProjectName.Core.Tests.MSTest/Tests_postaction.cs
//{[{
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.MSTest
{
    public class Tests
    {
        //^^
        //{[{

        // TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [TestMethod]
        public async void EnsureSampleDataServiceReturnsChartDataAsync()
        {
            var actual = await SampleDataService.GetChartDataAsync();

            Assert.AreNotEqual(0, actual.Count());
        }
        //}]}
    }
}
